api/src/main/java/org/openmrs/module/dhisconnector/api/impl/DHISConnectorServiceImpl.java

```diff
@@ -1371,6 +1371,15 @@ public String transformToDHISPeriod(Calendar startDate, Calendar endDate, String
 			if (lastRun == null || !(lastRan.get(Calendar.YEAR) + "W" + lastRan.get(Calendar.WEEK_OF_YEAR))
 			        .equals(endDate.get(Calendar.YEAR) + "W" + endDate.get(Calendar.WEEK_OF_YEAR)))
 				period = startDate.get(Calendar.YEAR) + "W" + startDate.get(Calendar.WEEK_OF_YEAR);
+		} else if (ReportingPeriodType.BiWeekly.name().equals(periodType)) {
+			startDate.add(Calendar.WEEK_OF_YEAR, -2);
```

Comment:
thanks @Piumal1999  do you need to reduce the date by two years?

Reply:
It's biweekly period, so we have to reduce 2 weeks to get the start date.(not 2 years)